src/modules/tests/test-status/test-status.actions.ts

```diff
@@ -1,30 +1,39 @@
 import { Action } from '@ngrx/store';
 
-// TODO: These actions are not relatable to an action in the app. Rename needed.
+export const SET_TEST_STATUS_BOOKED = '[???] Set Test Status to Booked';
+export const SET_TEST_STATUS_STARTED = '[???] Set Test Status to Started';
+export const SET_TEST_STATUS_DECIDED = '[???] Set Test Status to Decided';
+export const SET_TEST_STATUS_COMPLETED = '[???] Set Test Status to Completed';
+export const SET_TEST_STATUS_SUBMITTED = '[???] Set Test Status to Submitted';
```

Comment:
[???] Todo needs addressing.

Reply:
ahh thanks!

Reply:
Added the names of the components that dispatch these actions